### ISSUE TITLE
click-odoo-update: is checksum change detection in *.py necessary ? 

### ISSUE THREAD

**Issue by yassinelachgar** [acsone/click-odoo-contrib#83]
As far as I know, changes in .py files does not imply a module update for the changes to work. Would it not be optimal to detect changes only in XML files ?

The problem here is that a simple change in .py file triggers the module updates which takes additional time when running the server.

**Comment by sbidoul** [comment]
Yes it is necessary because python files contain models and field definition that need a database upgrade when changed.

**Comment by yassinelachgar** [comment]
Oh right, I totally missed that. But it would be nice if there was an option --ignore-py in case the changes dont contain models or fields definition, don't you think ?

**Comment by sbidoul** [comment]
There is already a system parameter in the database to control which files are part of the hash:

https://github.com/acsone/click-odoo-contrib/blob/d7f8c32a8cb162c16c3c470065485c0f3c467b7e/click_odoo_contrib/update.py#L26-L27

It could be interesting to control it with a CLI option too -- although it is a potential footgun :).

**Comment by yassinelachgar** [comment]
Merci stéphane 👍

**Comment by yassinelachgar** [comment]
If it's okay I'll work on a PR for this feature.

**Comment by sbidoul** [comment]
> If it's okay I'll work on a PR for this feature.

@yassinelachgar that is very welcome, thank you. I might be interesting to discuss the CLI option first (i.e. update the documentation in the README before coding). Also, when it comes to it, don't forget tests.
